### ISSUE TITLE
Updating handesk

### ISSUE THREAD

**Issue by ediblemanager** [BadChoice/handesk#110]
Thanks for your efforts on this - pretty much exactly what I was looking for!

I'm wondering how I would go about keeping handesk up to date once installed? If I make any changes to views etc., will they not be lost/create merge issues if I pull via git?

Not sure how difficult it would be to make this a composer package, which would simplify the process? Or some other route.

**Comment by BadChoice** [comment]
There is a deploy script that should take care of it all so just run .deploy.sh and it eill do the git pull, composer update and migrations



Jordi Puigdellívol

El 26 març 2018, a les 17:15, Gordon Thomson <notifications@github.com> va escriure:

**Comment by ediblemanager** [comment]
What would this do if you made a change to some of the views, and I also had made changes...wouldn't this result in a merge conflict?

**Comment by BadChoice** [comment]
Yes, in that case there would be a conflict, maybe we can try to do
something like “spark” does that checks the if the view is modified and
then it is not updated,
or do you have a better idea?

Jordi Puigdellívol

El 26 de marzo de 2018 a las 17:34:36, Gordon Thomson (
notifications@github.com) escribió:

What would this do if you made a change to some of the views, and I also
had made changes...wouldn't this result in a merge conflict?

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/BadChoice/handesk/issues/110#issuecomment-376209419>,
or mute the thread
